.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
         "no-unused-vars": ["error", { "vars": "all", "args": "none", "ignoreRestSiblings": true }],
         'one-var': ['error', 'never'],
         'padded-blocks': ['error', 'never'],
+        'prefer-const': ['error', {'destructuring': 'all'}],
```

Comment:
I think waiting to apply this until after everything was converted to es6 would have made this much easier to review lol

Reply:
We should've done this first, probably lol

It'd have shrunk the amount of lines to review quite a bit...

Reply:
But also, yeah, I could've done another PR with this :p